sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration/ConfigurationClientJavaDocCodeSnippets.java

```diff
@@ -275,45 +276,57 @@ public void deleteSetting() {
+        csStream.forEach(setting -> {
             System.out.printf("Key: %s, Value: %s", setting.key(), setting.value());
-        }
-        // END: com.azure.data.applicationconfig.configurationclient.listSettings#SettingSelector
+        });
+        // END: com.azure.data.applicationconfig.configurationclient.listSettings#settingSelector
+    }
 
-        /**
-         * Generates code sample for using {@link ConfigurationClient#listSettings(SettingSelector, Context)}
-         */
-        // BEGIN: com.azure.data.applicationconfig.configurationclient.listSettings#SettingSelector-Context
-        for (ConfigurationSetting setting : configurationClient.listSettings(new SettingSelector()
-            .keys("prodDBConnection"), new Context(key1, value1))) {
+    /**
+     * Generates code sample for using {@link ConfigurationClient#listSettings(SettingSelector, Context)}
+     */
+    public void listSettingsContext() {
+        ConfigurationClient configurationClient = createSyncConfigurationClient();
+        // BEGIN: com.azure.data.applicationconfig.configurationclient.listSettings#settingSelector-context
+        SettingSelector settingSelector = new SettingSelector().keys("prodDBConnection");
+        Context ctx = new Context(key2, value2);
+        PagedIterable<ConfigurationSetting> csStream =  configurationClient.listSettings(settingSelector, ctx);
+        csStream.forEach(setting -> {
             System.out.printf("Key: %s, Value: %s", setting.key(), setting.value());
-        }
-        // END: com.azure.data.applicationconfig.configurationclient.listSettings#SettingSelector-Context
+        });
+        // END: com.azure.data.applicationconfig.configurationclient.listSettings#settingSelector-context
     }
 
     /**
      * Generates code sample for using {@link ConfigurationClient#listSettingRevisions(SettingSelector)}
      */
     public void listSettingRevisions() {
         ConfigurationClient configurationClient = createSyncConfigurationClient();
-        // BEGIN: com.azure.data.applicationconfig.configurationclient.listSettingRevisions#SettingSelector
-        for (ConfigurationSetting revision : configurationClient.listSettingRevisions(new SettingSelector()
-            .keys("prodDBConnection"))) {
-            System.out.printf("Key: %s, Value: %s", revision.key(), revision.value());
-        }
-        // END: com.azure.data.applicationconfig.configurationclient.listSettingRevisions#SettingSelector
+        // BEGIN: com.azure.data.applicationconfig.configurationclient.listSettingRevisions#settingSelector
+        SettingSelector settingSelector = new SettingSelector().keys("prodDBConnection");
+        PagedIterable<ConfigurationSetting> csStream =  configurationClient.listSettingRevisions(settingSelector);
+        csStream.streamByPage().forEach(setting -> {
+            System.out.printf("Key: %s, Value: %s", setting.value().get(0).key(), setting.value().get(0).value());
```

Comment:
This isn't a good example. Try grabbing the page as a variable and doing something more relevant with it.